README.md

```diff
@@ -62,9 +70,56 @@ Outputs:
 
 * association summary statistics
 
-### To run
-
-Instructions to run each component of the pipeline using analysis runner are provided at the top of each script
+### SAIGE-QTL parameters explanation
+
+Clarifying the reasoning behind the parameters / flags used to run SAIGE-QTL.
+Most of these are (or will be) included in the official [documentation](https://weizhou0.github.io/SAIGE-QTL-doc/).
+
+Fit null model ([step 1](https://weizhou0.github.io/SAIGE-QTL-doc/docs/step1.html)):
+
+* ```pheno_file```: path specifying the location of the phenotype covariate file described above (build during part 2 of the pipeline)
+* ```cov_col_list```: string specifying the columns of the pheno_file that should be used as covariates (separated by a comma, no spaces)
+* ```sample_cov_col_list```: same as above, but specifying only, out of the columns above, the ones that are well defined at individual level (e.g., sex, age, ancestry PCs)
+* ```sample_id_pheno```: specify the column that represents individual IDs
+* ```output_prefix```: path to where the output files from step 1 (which will be read by step 2) should be written to
+* ```plink_path```: path to VRE plink files (specify just the prefix, but a .bim, .fam, and .bed files with the same prefix and in the same location should exist -- these are built in part 1)
+* ```pheno_col```: specify the column that should be used as phenotype, in our case the gene to test
+* ```trait_type```: specify the model to be used, ```count``` is the Poisson model which should be used here.
+* ```skip_vre```: boolean specifying whether the variance ratio estimation should be run or skipped, should always be false (Note that because the syntax is different between Python and R we encode this as the string ```FALSE``` instead of the boolean ```False```)
+* ```pheno_remove_zeros```: option to remove 0s from phenotype vector (default: ```FALSE``` as it does not make sense for the very sparse scRNA-seq data)
+* ```use_sparse_grm_null```: use sparse GRM to account for relatedness. This is implemented but would require a step0 in the pipeline to first construct this, which is not there at the moment (default: ```FALSE```)
+* ```use_grm_null```: same as above, but without the sparse option (default: ```FALSE```)
+* ```is_cov_offset```: set one of the covariates as offset (default: ```FALSE```)
```

Comment:
Which covariate gets set as an offset, and what does this mean/do?

Reply:
sorry that is not a good description, this actually is a boolean which adds an offset of ones if there are no covariates -- for us it is always false as we have plenty of covariates. I'll change this!